api/package-lock.json

```diff
@@ -1,4418 +1,8 @@
 {
   "name": "synergy",
   "version": "1.0.0",
-  "lockfileVersion": 2,
+  "lockfileVersion": 1,
   "requires": true,
-  "packages": {
```

Comment:
why did u remove all of these dependencies?

Reply:
nope, I didn't remove nothing )